server/server.go

```diff
@@ -844,6 +844,7 @@ func (s *Server) v0QueryPath(w http.ResponseWriter, r *http.Request, urlPath str
 	decisionID := s.generateDecisionID()
 
 	ctx := r.Context()
+	pretty := getBoolParam(r.URL, types.ParamPrettyV1, true)
```

Comment:
The default should not be true but false.

Also this is assigned more than 100 lines away from where it's used. I'd move this down to line 954.

Reply:
@anderseknert took reference from [v1DataPost](https://github.com/open-policy-agent/opa/blob/b4d5a234d0dd284b2f822a24791afd2d49756a42/server/server.go#L1364)
Seems like default is set to true wherever  `getBoolParam` is called, not sure why
Also in `v1DataPost` method, retrieving all details from `r http.Request` is done initially and then later utilised at different places, hence was thinking to have it consistent for v0 and v1 
However logically it does make sense to do both - set default as false and move assignment down the line

Reply:
Oh! I was wrong - it was not a default but "ifEmpty".. so if the parameter is provided _but_ without a value (like `/v0/data/test?pretty`) it should be counted as true.. so please leave it as it was then - it should still be moved though.

Reply:
done